db/operations_test.go

```diff
@@ -0,0 +1,199 @@
+package db_test
+
+import (
+	"github.com/luciodesimone/golang-bootcamp/beer"
+	"github.com/luciodesimone/golang-bootcamp/db"
+	"os"
+	"reflect"
+	"testing"
+)
+
+//we only use a copy of this variable to prevent data dependency
```

Comment:
If there is no data dependency between tests then every test case could use the `New(...)` method and create it's own beer.

No need for a global var that has the Beer.ID changing on every test.